torch/distributed/rpc/api.py

```diff
@@ -274,6 +274,7 @@ def _invoke_rpc(to, func, args=None, kwargs=None):
         raise TypeError("function should be callable.")
 
     qualified_name = torch.jit._find_builtin(func)
+    rf = torch.autograd._RecordFunction() if torch.autograd._profiler_enabled() else None
```

Comment:
Actually, I think that the `rf.before()` which starts the before callbacks which starts the CPU timer should be exposed to Python and called here probably, since this will allow us to include the Python serialization time. cc @pritamdamania87